test/CMakeLists.txt

```diff
@@ -18,15 +18,20 @@ list(APPEND BASIC_TESTS test/math.slim)
 list(APPEND BASIC_TESTS test/generic_ptrs1.slim)
 list(APPEND BASIC_TESTS test/generic_ptrs2.slim)
 
-list(APPEND BASIC_TESTS test/c/simple1.c)
-list(APPEND BASIC_TESTS test/c/simple2.c)
-list(APPEND BASIC_TESTS test/c/simple3.c)
-list(APPEND BASIC_TESTS test/c/simple4.c)
-list(APPEND BASIC_TESTS test/c/include.c)
+list(APPEND BASIC_C_TESTS test/c/simple1.c)
+list(APPEND BASIC_C_TESTS test/c/simple2.c)
+list(APPEND BASIC_C_TESTS test/c/simple3.c)
+list(APPEND BASIC_C_TESTS test/c/simple4.c)
+list(APPEND BASIC_C_TESTS test/c/include.c)
+list(APPEND BASIC_TESTS ${BASIC_C_TESTS})
 
 list(APPEND BASIC_TESTS samples/fib.slim)
 list(APPEND BASIC_TESTS samples/hello_world.slim)
 
 foreach(T IN LISTS BASIC_TESTS)
     add_test(NAME ${T} COMMAND slim ${PROJECT_SOURCE_DIR}/${T} -o test.spv)
 endforeach()
+
+if(NOT json-c_FOUND)
```

Comment:
This doesn't work, I think because the `find_package` happens in an unrelated subdirectory. I changed this to check for the existence of the target, and also whether clang is in the path